syncer/filter.go

```diff
@@ -58,26 +60,26 @@ func (s *Syncer) skipQuery(tables []*filter.Table, stmt ast.StmtNode, sql string
 	if len(tables) == 0 {
 		action, err := s.binlogFilter.Filter("", "", et, sql)
 		if err != nil {
-			return false, terror.Annotatef(terror.ErrSyncerUnitBinlogEventFilter.New(err.Error()), "skip query %s", sql)
+			return false, false, terror.Annotatef(terror.ErrSyncerUnitBinlogEventFilter.New(err.Error()), "skip query %s", sql)
 		}
 
 		if action == bf.Ignore {
-			return true, nil
+			return true, true, nil
```

Comment:
I'm not sure if this will cause us to track statements that shouldn't be tracked

For example, `truncate table` `rename table`